__specs__/adventureList.test.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import AdventuresList from '../src/components/AdventuresList';
+
+describe('AdventureList component functions as expected', () => {
+  const adventures = [{
```

Comment:
This is fine right now since its just one adventure in the array but keep in mind you can always move you test data to a separate file and then require it in all the test files where it's needed. Just something to think about if you ever find yourself wanting a larger amount of testing data.